### PR TITLE
January 2023 news

### PR DESCRIPTION
**Pull request by maelle**
TODOs

- [x] https://github.com/ropensci/phruta/pull/21

### CODE REVIEW

**Comment by mpadge** [comment]
@jeroen I just  added a bit in the Package Dev Corner on SystemRequirements (from current Line569 of `index.Rmd`) - can you please have a look, and edit as you like. Thanks!

File: content/blog/2023-01-20-ropensci-news-digest-january-2023/index.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+## Software :package:
+
+### New packages
+
+
+
+
+The following  package recently became a part of our software suite:
+
++ [phruta](https://docs.ropensci.org/phruta), developed by Cristian Roman Palacios: The phruta R package is designed to simplify the basic phylogenetic pipeline. Specifically, all code is run within the same program and data from intermediate steps are saved in independent folders. Furthrmore, all code is run within the same environment which increases the reproducibility of your analysis. phruta retrieves gene sequences, combines newly downloaded and local gene sequences, and performs sequence alignments. It has been [reviewed](https://github.com/ropensci/software-review/issues/458) by Anna Krystalli, Rayna Harris, and Frederick Boehm.
```
Comment (steffilazerte) [inline content/blog/2023-01-20-ropensci-news-digest-january-2023/index.md:63]: Not sure if you want to add this correction or not as it comes from the package itself...

```suggestion
+ [phruta](https://docs.ropensci.org/phruta), developed by Cristian Roman Palacios: The phruta R package is designed to simplify the basic phylogenetic pipeline. Specifically, all code is run within the same program and data from intermediate steps are saved in independent folders. Furthermore, all code is run within the same environment which increases the reproducibility of your analysis. phruta retrieves gene sequences, combines newly downloaded and local gene sequences, and performs sequence alignments. It has been [reviewed](https://github.com/ropensci/software-review/issues/458) by Anna Krystalli, Rayna Harris, and Frederick Boehm.
```

**Review by steffilazerte — COMMENTED** [review]
Looks good!
I added the coworking edits. Just a couple of comments other than that. 
Thanks!

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [X] publication date is ok
* [X] alternative text of images is informative
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
* [X] YAML field 'preface' is present if necessary

File: content/blog/2022-12-16-ropensci-news-digest-december-2022/index.Rmd [hunk 2]
```diff
@@ -560,6 +560,11 @@ Explore [other use cases](/usecases) and [report your own](https://discuss.ropen
 
 * **[onekp](https://docs.ropensci.org/onekp), Access sequences from the 1000 Plant Initiative (1KP)**. The 1000 Plants Initiative (www.onekp.com) has sequenced the transcriptomes of over 1000 plant species. This package allows these sequences and metadata to be retrieved and filtered by code, species or recursively by clade. Scientific names and NCBI taxonomy IDs are both supported. [Issue for volunteering](https://github.com/ropensci/onekp/issues/9) 
 
+- **[citesdb](https://docs.ropensci.org/citesdb/)**,a high-performance database of shipment-level CITES trade data. Provides convenient access to over 40 years and 20 million records of
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2022-12-16-ropensci-news-digest-december-2022/index.Rmd:563]: no it's a mistake, thank you!!

File: content/blog/2023-01-20-ropensci-news-digest-january-2023/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,573 @@
...
+
+Refer to our [recent blog post](/blog/2022/10/17/maintain-or-co-maintain-an-ropensci-package/#packages-looking-for-co-maintainers) to identify packages where help is especially wished for!
+
+## Package development corner
+
+Some useful tips for R package developers. :eyes:
+
+### Bad code? Good code?
+
+Why not feel code shame when looking at older, less good code of yours?
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2023-01-20-ropensci-news-digest-january-2023/index.Rmd:556]: Right?! such a good post

File: content/blog/2023-01-20-ropensci-news-digest-january-2023/index.md (hunk 1 above)
Comment (maelle) [reply content/blog/2023-01-20-ropensci-news-digest-january-2023/index.md]: fixed in the package source, thank you!

**Pull request merged**